common/interpreter/wawaka_wasm/CMakeLists.txt

```diff
@@ -69,6 +69,11 @@ if (NOT DEFINED WAMR_BUILD_LIBC_WASI)
   set (WAMR_BUILD_LIBC_WASI 0)
 endif ()
 
+if (NOT DEFINED WAMR_BUILD_FAST_INTERP)
```

Comment:
Is fast interpreter unstable? if not, maybe mention in README? Or is this a standard WAMR variable documented already there?

BTW: should we test also with WAMR_BUILD_FAST_INTERP=1?

Reply:
Ah, I never switched the fast interpreter back on after I was testing with AoT enabled. I'll turn it on since we're still running interpreted WASM here.

Reply:
This variable is already documented in the WAMR code base. The claim is that the fast interpreter is about 2x faster, but consumes 2x more memory. We can mention this trade-off in our documentation, and decide if we want the default to be fast or "classic" WAMR.

Reply:
ah, ok. thanks for clarification. must have been in a previous pr which i must have missed or simply already forgotten again :-)

Reply:
Just to clarify some more, this `WAMR_BUILD_FAST_INTERP` variable _is_ a recent addition to WAMR added since our last PR, so I don't think you missed anything :)

Reply:
Oops, i misread `We can mention` as `We do mention` in your reply. Maybe i shouldn't try to multi-task four different things ;-)